docs/sources/developers/plugins/create-a-grafana-plugin/develop-a-plugin/build-a-panel-plugin-with-d3.md

```diff
@@ -1,14 +1,7 @@
 ---
 title: Build a panel plugin with D3.js
-summary: Learn how to use D3.js in your panel plugins.
 description: how to use D3.js in your panel plugins.
-id: build-a-panel-plugin-with-d3
-categories: ['plugins']
-tags: ['beginner']
-status: Published
-authors: ['grafana_labs']
-Feedback Link: https://github.com/grafana/tutorials/issues/new
-weight: 60
+weight: 200
```

Comment:
a lot of these are missing the keywords metadata and maybe others, is that to be added later?

Reply:
I've gone ahead and added them now.